### DISCUSSION TITLE
Configure Suggestion Appearance

### DISCUSSION THREAD

**Discussion by Erix4** [community/community#7234] [Copilot Conversations] [answered]
I use the Dracula theme in vscode, and it can be difficult to tell my comments and the code suggestions apart. It would be nice to be able to change their appearance to make this more clear.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Hey, you can customize the way we display the inline suggestion by putting the following settings in the settings.json:
```
"workbench.colorCustomizations": {
        "editorGhostText.border": "#ff0000",
        "editorGhostText.foreground": "#ff0000",
    }
```

**Reply by Erix4** [reply]
It works! Thanks! Also, this addon is amazing

**Reply by MulleDK19** [reply]
Any other secret settings?
